build/opt-build-sys-dependencies.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+set -xe
+
+# Add python ppa
+add-apt-repository -y ppa:deadsnakes/ppa
+# Add R key
+echo "deb http://cran.rstudio.com/bin/linux/ubuntu xenial/" | tee -a /etc/apt/sources.list
+gpg --keyserver keyserver.ubuntu.com --recv-key E084DAB9
+gpg -a --export E084DAB9 | apt-key add -
+
+# install python3, R, bedtools, bowtie1, bowtie2, blast, gmap and other packages for RSeQC to install
+apt-get update
+apt-get install -yq --no-install-recommends \
+python3.7 python3.7-dev \
+r-base r-base-dev \
+bedtools=${VER_BEDTOOLS} \
+bowtie=${VER_BOWTIE1} \
+bowtie2=${VER_BOWTIE2} \
+ncbi-blast+=${VER_BLAST} \
+gmap=${VER_GMAP} \
+libcurl4-gnutls-dev zlib1g-dev
+apt-get upgrade -yq gcc
+
+# install R packages:
+Rscript -e 'install.packages("ada")'  # required by Defuse
+
+# Replace python3
```

Comment:
See [here](https://jcutrer.com/linux/upgrade-python37-ubuntu1810) for correct way to do this.

Reply:
Thanks, now it's done properly.